cst/plugins/eu.esdihumboldt.cst.functions.collector/plugin.xml

```diff
@@ -15,7 +15,6 @@
       <propertyFunction
             category="eu.esdihumboldt.hale.align.general"
             cellExplanation="eu.esdihumboldt.cst.functions.collector.explanations.AssignFromCollectorExplanation"
-            icon="icons/augmentation.gif"
```

Comment:
I removed this because the icon does not exist in the bundle and this produced a warning. If the function is an augmentation (no source) it will have that icon by default.
If we come up with a better fitting icon we should add it, but I think that has not the highest priority.